book/src/remapping.md

```diff
@@ -110,3 +109,12 @@ Special keys are encoded as follows:
 | Escape       | `"esc"`        |
 
 Keys can be disabled by binding them to the `no_op` command.
+
+All other keys such as `?`, `!`, `-` etc. can be used literally:
+
+```toml
+[keys.normal]
+"?" = ":write"
+"!" = ":write"
+"-" = ":write"
+```
```

Comment:
I would also mention here that `-` can't be used when combined with a modifier, for example <kbd>Alt</kbd>+<kbd>-</kbd> should be written as `A-minus`: `A--` isn't accepted.

Reply:
Added